sublime/pythonLiveCoding.py

```diff
@@ -0,0 +1,303 @@
+import io
+import logging
+import functools
+import subprocess
+
+import sublime, sublime_plugin
+
+
+XMIN, YMIN, XMAX, YMAX = list(range(4))
+OUTPUT_VIEW_ID = 'output_view_id'
+INPUT_VIEW_TRACER_ARGS = 'input_view_tracer_args'
+CANVAS_START = 'start_canvas'
+CANVAS_END = 'end_canvas'
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+def find_view(view_id):
+    for window in sublime.windows():
+        for view in window.views():
+            if view.id() == view_id:
+                return view
+    return None
+
+
+def trace_code(input, pargs=None):
+
+    # Pull location of python exe and code_tracer.py script from user settings.
+    settings = sublime.load_settings('PythonLiveCoding.sublime-settings')
+    py_path = settings.get('python_executable')
+    tracer_path = settings.get('code_tracer')
+    args = [py_path, tracer_path]
+    if pargs is not None:
+        args.extend(pargs)
+    args.append('-')
+
+    # Startup info so we don't open a new command prompt.
+    startupinfo = subprocess.STARTUPINFO()
```

Comment:
According to the [Python docs]:

>The STARTUPINFO class and following constants are only available on Windows.

Would you like me to add some commits to make this work on GNU/Linux?

[Python docs]: https://docs.python.org/3/library/subprocess.html#windows-popen-helpers

Reply:
Fixed as per https://stackoverflow.com/questions/1016384/cross-platform-subprocess-with-hidden-window